src/todolist.js

```diff
@@ -0,0 +1,87 @@
+import DeleteIcon from './delete_black_24dp.svg';
+
+const toDoContainer = document.getElementById('todo-container');
+export default class ToDoList {
+  createEmpty = () => {
+    const todoes = [];
+    localStorage.setItem('todoes', JSON.stringify(todoes));
+  };
+
```

Comment:
- [ ] Kindly add a function for editing task descriptions, as the requirements say `Implement a function for editing task descriptions`.